ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -271,6 +271,18 @@ open class BadgeView: UIView {
         }
     }
 
+    /**
+     The maximum allowed size point for the label's font. This property can be used
+     to restrict the largest size of the label when scaling due to Dynamic Type. The
+     default value is 0, indicating there is no maximum size.
+     */
+    open var maxFontSize: CGFloat = 0 {
```

Comment:
Is this going to be needed in ObjC?

Reply:
based on my understanding on this client usage, it seems like they are using swift. so i purposely didn't expose objc until i have to.

Reply:
Thanks for confirming!